bluemira/base/reactor.py

```diff
@@ -110,10 +111,29 @@ def show_cad(self, dim: str = "xyz", **kwargs):
             The dimension of the reactor to show, typically one of
             'xz', 'xy', or 'xyz'. (default: 'xyz')
         """
-        if dim not in _PLOT_DIMS:
-            raise ReactorError(
-                f"Invalid plotting dimension '{dim}'. "
-                f"Must be one of {str(_PLOT_DIMS)[1:-1]}"
+        # give dims_to_show a default value
+        dims_to_show = ("xyz",) if len(dims) == 0 else dims
+
+        # if a kw "dim" is given, it is only used
+        if kw_dim := kwargs.pop("dim", None):
+            warn(
+                "Using kwarg 'dim' is no longer supported. Simply pass in the dimensions you would like to show, e.g. show_cad('xz')",
+                category=DeprecationWarning,
```

Comment:
Just so we have a shorter line, autojoining strings
```suggestion
        if kw_dim := kwargs.pop("dim", None):
            warn(
                "Using kwarg 'dim' is no longer supported. "
                "Simply pass in the dimensions you would like to show, e.g. show_cad('xz')",
                category=DeprecationWarning,
```